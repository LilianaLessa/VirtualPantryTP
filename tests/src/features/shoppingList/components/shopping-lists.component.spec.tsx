// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-nocheck
import * as React from "react";
import * as renderer from "react-test-renderer";
import ShoppingLists from "../../../../../src/features/shoppingList/components/shopping-lists.component";

describe("App", () => {
  it("snapshot test", () => {
    const tree = renderer.create(<ShoppingLists shoppingLists={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
