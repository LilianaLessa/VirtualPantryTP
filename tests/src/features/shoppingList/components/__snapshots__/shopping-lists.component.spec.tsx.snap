// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View>
  <Text>
    No shopping lists saved yet.
  </Text>
</View>
`;
